.pre-commit-config.yaml

```diff
@@ -1,6 +1,19 @@
 repos:
--   repo: https://github.com/awslabs/cfn-python-lint
-    rev: v0.23.3
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/adrienverge/yamllint
+    rev: v1.23.0
+    hooks:
+    -   id: yamllint
+-   repo: https://github.com/aws-cloudformation/cfn-python-lint
+    rev: v0.31.1
     hooks:
     -   id: cfn-python-lint
-        files: templates/.*\.(json|yml|yaml)$
+        files: ec2/.*\.(json|yml|yaml)$
```

Comment:
I could not find a way to get cfn-python-lint to scan all CFN files.  When i set to this to `.*\.(json|yml|yaml)$` it causes `git commit` to fail with 
```
AWS CloudFormation Linter................................................Failed
- hook id: cfn-python-lint
- exit code: 2

E1001 Top level item repos isn't valid
None:1:1

E1001 Missing top level item Resources to file module
None:1:1
```

Reply:
@zaro0508 this works for me: `.*/.*\.(json|yml|yaml)$`

Reply:
thanks @tthyer